### DISCUSSION TITLE
Split View is currently not supported

### DISCUSSION THREAD

**Discussion by Pigeonwingattack** [keiwando/toonsquid#483] [Ideas]
I’ve been using toonsquid a lot these last 2 weeks and I’m noticing that the main thing that hinders progress is the inability of split view. 

Even if using sometimes the ipad in front of a computer, while drawing on other apps for example, I still use split view. And for animation it really is needed to check some references for example (images, Vizeef app, etc). The option to have a window floating doesnt help much because it hides a lot of options.

I have no idea how difficult it is to make the app support this, but putting a post up since i didnt seem to find anything mentioning it yet :)

**Comment by amoraleite** [comment]
I would like very much this feature! And would be incredibly if air display turn the ToonSquid a second monitor for hold the timeline, you know, move the time line for another screen…

**Comment by DWrightAnimation** [comment]
The only way Split View could be possible is if ToonSquid got the ability to do vertical UI.
